torch/onnx/utils.py

```diff
@@ -750,6 +745,12 @@ def _export(model, args, f, export_params=True, verbose=False, training=None,
                         opened_file.write(v)
             else:
                 raise RuntimeError("Unknown export type")
+
+            if operator_export_type is OperatorExportTypes.ONNX and \
```

Comment:
Since it's important that this happens after some other code in this function, please add a a unit test ensuring that is the case. Whenever you make a change to the functionality of a public function without making a corresponding change to the tests, that suggests we are missing some test coverage.